XCalendar.Maui/Converters/LocalizeDayOfWeekAndCharLimitConverter.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Globalization;
+using XCalendar.Maui.Resources;
+using XCalendar.Core.Extensions;
+
+namespace XCalendar.Maui.Converters
+{
+    public class LocalizeDayOfWeekAndCharLimitConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (value is not DayOfWeek dayOfWeek)
+            {
+                return string.Empty;
+            }
+
+            string resourceName = $"{dayOfWeek}";
+
+            string dayName = XCalendarResource.ResourceManager.GetString(resourceName, culture);
```

Comment:
Instead of managing translations ourselves, .NET provides a built-in method to do this:
https://learn.microsoft.com/en-us/dotnet/api/system.globalization.datetimeformatinfo.getdayname?view=net-7.0

```suggestion
            string dayName = culture.DateTimeFormat.GetDayName(dayOfWeek);
```

Reply:
AMAZING, I did not know this.

Reply:
Ikr, me neither I just checked because I thought since there is culture-specific formatting for `DateTime` and such, there could be the same for `DayOfWeek`.